src/core/qgslegendsettings.h

```diff
@@ -178,6 +178,20 @@ class CORE_EXPORT QgsLegendSettings
      */
     void setMapScale( double scale ) { mMapScale = scale; }
 
+    /**
+     * Returns the factor of map units per pixel for symbols with size given in map units calculated by dpi and mmPerMapUnit
+     * \see setMapUnitsPerPixel()
+     * \since QGIS 3.8
+     */
+    double mapUnitsPerPixel() const;
+
+    /**
+     * Sets the the mmPerMapUnit calculated by \a mapUnitsPerPixel mostly taken from the map settings.
```

Comment:
`the the` -> `the`

I just made a PR to fix master documentation accordingly (https://github.com/qgis/QGIS/pull/9288).